opentasks/src/main/java/org/dmfs/tasks/model/constraints/UpdateAllDay.java

```diff
@@ -58,11 +58,6 @@ public Boolean apply(ContentSet currentValues, Boolean oldValue, Boolean newValu
                 }
 
             }
-            else if ((newValue == null || !newValue) && oldValue != null && oldValue)
```

Comment:
same here, the comment is still a valid issue (maybe not a big one though) and we should keep it for reference/as a reminder.